src/App.js

```diff
@@ -36,19 +39,28 @@ const donations = [
   },
 ];
 
-function App() {
+export default function App() {
   return (
-    <>
+    
+    <div>
       <TopBar />
       <main className="container">
-        <section className="sidebar">{/* Recent Donations */}</section>
-        <section className="">
+        
+        <section className="sidebar">
+          <RecentDonations donations={donations}/>
+          </section>
+          <section className="">
           {/* Progress */}
           {/* Donation Form */}
+          <Progress max={targetAmount} />
```

Comment:
for progress you need the target amount and the total amount of donations already made. What variable holds that information? You would need to pass that as a prop too.